.kokoro/run/filesystem.cfg

```diff
@@ -0,0 +1,7 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Set the folder in which the tests are run
+env_vars: {
```

Comment:
This shouldn't be need if we are using GitHub Actions. Not all sample tests have been migrated off of Kokoro.